src/OpenSage.Game/Logic/Object/GameObject.cs

```diff
@@ -138,6 +136,9 @@ internal set
         }
     }
 
+    // TODO: Remove this. There should be a better way to access the game context.
+    internal GameContext Context => _gameContext;
```

Comment:
This looks so familiar - I thought we actually removed this "recently"? What's using it now?

Reply:
I needed it [here](https://github.com/OpenSAGE/OpenSAGE/blob/377ffc35945bfdecebb0f485e7a5830b1bc02afd/src/OpenSage.Game/Logic/Object/Update/UpdateModule.cs#L63) and [here](https://github.com/OpenSAGE/OpenSAGE/blob/physicsbehavior/src/OpenSage.Game/Logic/Object/Behaviors/PhysicsBehavior.cs#L662). But you're right, I could propagate `GameContext` to the base `ObjectModule` class, so I'll do that.

Reply:
Done.